RedHat-UX/red-hat-design-system
# feat(spinner): Added styling for a slotted <p>

## What I did

1. Added styles for `::slotted(p)`


## Testing Instructions

Check [deploy preview](https://deploy-preview-776--red-hat-design-system.netlify.app/components/spinner/demo/)

## Notes to Reviewers
